src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -1935,6 +1937,14 @@ input Modes {
   transportModes: [TransportModes]
 }
 
+"Defines one point which the journey must pass through."
+input PassThroughPoint {
+  "Optional name of the pass-through point for debugging and logging. It is not used in routing."
+  name: String
+  "The ids of elements in the OTP model. Defines one or multiple stops of the pass-through point, of which only one is required to be passed. Currently supports Quay, StopPlace, multimodal StopPlace, and GroupOfStopPlaces."
+  placeIds: [String!]
```

Comment:
```suggestion
  "The list of *stop location ids* witch define the pass-through point. At least one id is required.
  Quay, StopPlace, multimodal StopPlace, and GroupOfStopPlaces are supported location types. 
  The journey must pass through at least one of these entities - not all of them."
  placeIds: [String!]
```

Reply:
PS! This need to be added in the schema definition/code not here...

Reply:
Much better.